docs/architecture.md

```diff
@@ -0,0 +1,40 @@
+# Architecture
+
+We have a classic nuxt architecture. What you should know here is our components folder:
```

Comment:
Can you provide a link to some docs defining what "a classic nuxt architecture" is?
I've never worked with nuxt, so I don't know what is the standard architecture.

Reply:
just the basic directory structure and add the vue design system pattern for our components

https://nuxtjs.org/guide/directory-structure